src/redux/categories/categories.js

```diff
@@ -0,0 +1,19 @@
+const categoriesReducer = (state = [], action) => {
+  switch (action.type) {
+    case 'STATUS_INFO':
+      return [
+        ...state, action.payload,
+      ];
+    default:
+      return state;
+  }
+};
```

Comment:
- [ ] I think you should define `STATUS_INFO` as an action. The value of the action type needs to follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux), so it can be something like: `const STATUS_INFO = bookstore/categories/STATUS_INFO` 🙃 🙃